src/dlsproto/node/neo/request/GetRange.d

```diff
@@ -266,7 +266,10 @@ public abstract scope class GetRangeProtocol_v1
 
             if(!got_next)
```

Comment:
Maybe add a comment here: "No more data in the range. Ending request." (or something).

Reply:
Added few comments.